<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gNAP.qrc">
    <normaloff>:/picture/picture/icon.png</normaloff>:/picture/picture/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#background{border-image: url(:/picture/picture/background.jpg);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Do">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>350</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Do{
        font: 75 36pt &quot;Aharoni&quot;;
        color:white;
}</string>
    </property>
    <property name="text">
     <string>Do</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="TF_TF">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#TF_TF{
        border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;
}

#TF_TF:hover{
        border-image: url(:/picture/picture/green2.png);
}
</string>
    </property>
    <property name="text">
     <string>TF-TF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TF_Gene">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#TF_Gene{
        border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;
}

#TF_Gene:hover{
        border-image: url(:/picture/picture/green2.png);
}
</string>
    </property>
    <property name="text">
     <string>TF-Gene</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Gene_Info">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#Gene_Info{
        border-image: url(:/picture/picture/green1.png);
        color:white;
        font: 75 20pt &quot;Aharoni&quot;;
}

#Gene_Info:hover{
        border-image: url(:/picture/picture/green2.png);
}
</string>
    </property>
    <property name="text">
     <string>Gene Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TU_Info">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#TU_Info{
        border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;
}

#TU_Info:hover{
        border-image: url(:/picture/picture/green2.png);
}
</string>
    </property>
    <property name="text">
     <string>TU Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Promoter_Info">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#Promoter_Info{
        border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;
}

#Promoter_Info:hover{
        border-image: url(:/picture/picture/green2.png);
}
</string>
    </property>
    <property name="text">
     <string>Promoter Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#close{
        border-image: url(:/picture/picture/close1.png);
}

#close:hover{

        border-image: url(:/picture/picture/close2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Take_a_nap">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#Take_a_nap{
        border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;
}

#Take_a_nap:hover{
        border-image: url(:/picture/picture/green2.png);
}
</string>
    </property>
    <property name="text">
     <string>Take a gNAP!</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>350</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="doing">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>350</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#doing{font: 13pt &quot;Courier10 BT&quot;;
color:white;}</string>
    </property>
    <property name="text">
     <string>Doing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar_start">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#progress_bar_start{
        border-image: url(:/picture/picture/progress.png);
}
#progress_bar_start::chunk{
     background-color: white;
     width: 2px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="promoter">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#promoter{
        font: 16pt &quot;Gisha&quot;;
}</string>
    </property>
    <property name="text">
     <string>Copy Promoter Sequ. Here!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Promoter_done">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#Promoter_done{border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;}
#Promoter_done:hover{
        border-image: url(:/picture/picture/green2.png);
}</string>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gene">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#gene{
        font: 16pt &quot;Gisha&quot;;
}</string>
    </property>
    <property name="text">
     <string>Copy Gene Sequ. Here!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Gene_done">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#Gene_done{border-image: url(:/picture/picture/green1.png);
color:white;
        font: 75 20pt &quot;Aharoni&quot;;}
#Gene_done:hover{
        border-image: url(:/picture/picture/green2.png);
}</string>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gNAP.qrc"/>
 </resources>
 <connections/>
</ui>
